stl/inc/array

```diff
@@ -617,8 +617,10 @@ public:
     _DEPRECATE_TR1_NAMESPACE void assign(const _Ty&) {}
 #endif // _HAS_TR1_NAMESPACE
 
+    template <class _OTy = _Ty, enable_if_t<!is_const_v<_OTy>, int> = 0>

```

Comment:
1. I don't think we want `fill` to SFINAE away: typically with named (non-operator) functions that don't participate in concepts or traits it's better to provide a clear error message with `static_assert` than to have the compiler tell the user there's no such function.

2. `is_const_v` is the wrong test, because there are `const T`s that are fillable and/or swappable (https://godbolt.org/z/5n4x8j). I think we want to `static_assert` `is_copy_assignable_v<_Ty>` here.